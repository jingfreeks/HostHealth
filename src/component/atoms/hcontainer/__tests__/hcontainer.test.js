import React from 'react';
import {render} from '@testing-library/react-native';
import Button from '../hcontainer';

describe('HContainer Component', () => {
  it('Should work as expected to get snapshot', () => {
    const all = render(<Button onPress={() => jest.fn()} />);
    expect(all.toJSON()).toMatchSnapshot();
  });
});
