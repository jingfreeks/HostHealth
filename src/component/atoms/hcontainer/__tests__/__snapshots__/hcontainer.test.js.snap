// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HContainer Component Should work as expected to get snapshot 1`] = `
<View
  style={
    [
      {
        "flexDirection": "row",
      },
    ]
  }
/>
`;
